### ISSUE TITLE
Configure Content Security Policy (CSP)

### ISSUE THREAD

**Issue by Juanlu001** [QuantStack/voila#29]
I was trying to access voila from a non-localhost IP and I was getting some CSP violations, so following @carreau's advice [here](https://github.com/jupyter/notebook/issues/886#issuecomment-166291876) I added the following configuration:

```patch
--- /home/juanlu/.local/share/virtualenvs/poseidon-oYfLlCwx/src/voila/voila/app.py.old	2018-11-28 01:31:54.073333937 +0100
+++ /home/juanlu/.local/share/virtualenvs/poseidon-oYfLlCwx/src/voila/voila/app.py	2018-11-28 01:32:45.897266756 +0100
@@ -141,8 +141,7 @@
             jinja2_env=env,
             static_path='/',
             server_root_dir='/',
-            contents_manager=contents_manager
+            contents_manager=contents_manager,
+            headers={'Content-Security-Policy': "frame-ancestors 'self' http://host.eu"},
         )
 
         base_url = webapp.settings.get('base_url', '/')
```

Is there a better way to do it?

**Comment by maartenbreddels** [comment]
Not for the moment, I think we should have a similar command line option though. Actually, we should think how we can reuse some of the notebook command line options.

**Comment by SylvainCorlay** [comment]
This should already work when using as a notebook or jupyter_server extension.

Exposing these to the voila standalone app should not be difficult but it is not in yet.

**Comment by timkpaine** [comment]
@Juanlu001 @SylvainCorlay @maartenbreddels WIP in https://github.com/QuantStack/voila/pull/89, tests failing though, will work on it

**Comment by timkpaine** [comment]
@Juanlu001 can you check if this is fixed by https://github.com/QuantStack/voila/pull/69

**Comment by Juanlu001** [comment]
Sorry, not working with voila at the moment :frowning_face:

**Comment by maartenbreddels** [comment]
This should be fixed by #69 indeed, if not, please reopen.
